pkg/integrations/github/snippet-windows.ps1

```diff
@@ -0,0 +1,5 @@
+New-Item -Path C:\actions-runner -Type Directory ; cd C:\actions-runner
+Invoke-WebRequest -Uri {{ .RunnerURL }} -OutFile actions-runner-win.zip
+Add-Type -AssemblyName System.IO.Compression.FileSystem ;
+[System.IO.Compression.ZipFile]::ExtractToDirectory("$PWD\actions-runner-win.zip", "$PWD")
+./config.cmd --token $ghToken --url {{ .RepoURL }} --name {{ .Name }} --unattended --runasservice --replace --labels {{ .Labels }}
```

Comment:
Did you check if tihs starts the cli as the current user and not as root?

Reply:
yes, this starts the service as the normal user on windows